nss-day-cohort-41/nutshell-my-pretty-rose-tree
# Js event list

# Description
Filtered and displayed events by active user, sorted events displaying latest date first and added delete functionality on each event.

## Type of change
Please delete options that are not relevant.

# Testing Instructions
No events should appear if new user. Click create an event and form fields will populate. Click submit event and new event will be displayed in list. The latest date should be displayed first in the list. Click delete to remove event from list.

# Checklist:
